### PR TITLE
Prepare August patch release for FR and MA

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 3.2.0-beta.1 (Unreleased)
-
+## 3.1.3 (2021-08-11)
+#### Dependency Updates
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:4]:
```suggestion
### Dependency Updates
```

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.0.1 (2021-08-11)
+#### Dependency Updates
```
Comment (mssfang) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:4]:
```suggestion
### Dependency Updates
```

**Review by mssfang — APPROVED** [review]

**Comment by mssfang** [comment]
You should release the patch version from a release branch.

**Comment by samvaity** [comment]
>You should release the patch version from a release branch.

No ongoing beta development currently.

**Pull request merged**